app/views/users/_form.html.erb

```diff
@@ -0,0 +1,12 @@
+<%= form_for @user do |f| %>
```

Comment:
In cho anh params của form_for, form_tag, form_with và so sánh giữa chúng

Reply:
form_for:
"user"=>{"name"=>"Tạ Duy Hiếu AAA", "email"=>"taduyhieucntt98@gmail.com", "password"=>"", "password_confirmation"=>""}

form_tag:
"name"=>"hieu", "email"=>"hieu@gmail.com", "password"=>"", "password_confirmation"=>""
Em thấy, form_for với form_with thì các thuộc tính của user nằm trong đối tượng user, còn form_tag thì các thuộc tính user nằm ngay trong params, không được đóng gói